spec/requests/api/providers_spec.rb

```diff
@@ -1116,4 +1116,14 @@ def gen_import_request
       end
     end
   end
+
+  context "#options" do
+    it "returns options for queried provider class" do
+      api_basic_authorize
+      ExtManagementSystem.descendants.select { |ems| ems.respond_to?(:provider_settings) }.each do |ems_type|
```

Comment:
Your test logic here mirrors that of the implementation, so I'm wondering what value this test is providing? IMO, using literals to represent expected values is much clearer and will provide useful feedback in the case of breakages or false assumptions about the implementations.